dozer/cogs/levels.py

```diff
@@ -44,9 +44,12 @@ def total_xp_for_level(level):
         # https://github.com/Mee6/Mee6-documentation/blob/9d98a8fe8ab494fd85ec27750592fc9f8ef82472/docs/levels_xp.md
         # > The formula to calculate how many xp you need for the next level is 5 * (lvl ^ 2) + 50 * lvl + 100 with
         # > your current level as lvl
+        if level >= 1000000:  # If the level gets too big, dozer will hang trying to calculate the level. A better way needs to be found to calculate
+            DOZER_LOGGER.warning("Member XP exceeded maximum calculation limit")  # level's but this is the best for now
```

Comment:
This should be a critical error, not a warning, as this is breaking functionality of the bot.